guide/src/logging.md

```diff
@@ -0,0 +1,60 @@
+# Logging
+
+It is desirable if both the Python and Rust parts of the application end up
+logging using the same configuration into the same place.
+
+This can trivially be accomplished by not logging at all in one of them. If
+that's not a good enough solution, it is possible to install a logger that sends
+the log messages to the counterpart.
+
+# The Rust -> Python direction
```

Comment:
I wonder if this title would be even stronger if it said what the reader will learn in this section:

```suggestion
## Using `pyo3-log` to send Rust log messages to Python
```

Also I would prefer only have one top level `#` heading so I added an extra `#` in the suggestion.